### DISCUSSION TITLE
Upgrading ASReview lab during a project

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1872] [Q&A] [answered]
Hello,  

my computer says there is a new update: 1.6.3 version.
I am in the middle of doing a review with 27,000 plus records. I am wondering if it will couse problems to upgrade the software at this stage.

**Comment by Rensvandeschoot** [comment, marked as answer]
You can check out what's new in version 1.6.3 on our [release overview page](https://github.com/asreview/asreview/releases). Upgrading should not impact your current project, as we aim to maintain backward compatibility with all updates before version 2.0. However, to be absolutely certain, we recommend creating a backup of your project. You can back up your project by [exporting](https://asreview.readthedocs.io/en/latest/manage.html) it and storing the export file in a secure location. This is always a good practice, especially if your project folder isn't in a location that is automatically backed up
